astro.config.mjs

```diff
@@ -105,5 +105,6 @@ export default defineConfig({
     '/about/team': '/about',
     '/about/tsc': '/about',
     // '/learning-hub/[...slug]': '/about',
+    '/whitepapers/protecting-investment-in-code-optimisation-with-toolchain-ci-linaro-white-paper/': '/whitepapers',
```

Comment:
should this not link to the actual  whitepaper https://static.linaro.org/assets/automotive_white_paper_0921.pdf ?

Reply:
https://static.linaro.org/assets/Protecting-Investment-in-Code-Optimisation-with-Toolchain-CI-Linaro-White-Paper.pdf

Reply:
https://www.linaro.org/whitepapers the link here is currently wrong as well

Reply:
I did think about that but, as I pointed out to Chiara on the Jira issue, I felt it might be better to land on the whitepapers page itself so that they can see what other whitepapers are there.

On the old site, that URL lands on a page with a button that you click to then see the whitepaper so it is (somewhat) consistent from that perspective.

If Chiara objects, I can make the change to redirect to the PDF.